lib/src/Uncertainty/Distribution/PointConditionalDistribution.cxx

```diff
@@ -251,6 +335,122 @@ void PointConditionalDistribution::update()
+    OptimizationProblem problemU(objectiveU);
+    problemU.setMinimization(false);
+    problemU.setBounds(bounds);
+    OptimizationAlgorithm algo(OptimizationAlgorithm::GetByName(ResourceMap::GetAsString("PointConditionalDistribution-OptimizationAlgorithm")));
+    algo.setProblem(problemU);
+    algo.setStartingPoint(start);
+    algo.run();
+    supU_ = std::exp(algo.getResult().getOptimalValue()[0]);
+    LOGDEBUG(OSS() << "supU_=" << supU_ << " u*=" << algo.getResult().getOptimalPoint());
+
+    // Second, the lower and upper bounds on V
+    const Function objectiveV(new PointConditionalDistributionVBoundEvaluation(*this, r_));
+    infV_.resize(dimension);
+    supV_.resize(dimension);
+    const Point zero(dimension, 0.0);
+    for (UnsignedInteger i = 0; i < dimension; ++ i)
+    {
+      const Function objectiveVI(objectiveV.getMarginal(i));
+      OptimizationProblem problemVI(objectiveVI);
+      problemVI.setMinimization(false);
+      if (ub[i] > 0.0)
+      {
+        // find a feasible starting point in [0, ub]
+        start.clear();
+        for (UnsignedInteger k = 0; k < candidateNumber; ++ k)
+        {
+          Point candidate(sequence.generate());
+          for (UnsignedInteger j = 0; j < dimension; ++ j)
+            candidate[j] = candidate[j] * ub[j];
+          if (computePDF(candidate) > 0.0)
+          {
+            start = candidate;
+            break;
+          }
+        }
+        if (!start.getDimension())
+          throw InternalException(HERE) << "Could not find a feasible starting point to initialize ration of uniforms V sup";
+        problemVI.setBounds(Interval(zero, ub));
+        algo.setProblem(problemVI);
+        algo.setStartingPoint(start);
+        algo.run();
+        supV_[i] = std::exp(algo.getResult().getOptimalValue()[0]);
+        LOGDEBUG(OSS() << "supV_[" << i << "]=" << supV_[i] << " v*=" << algo.getResult().getOptimalPoint());
+      }
+      if (lb[i] < 0.0)
+      {
+        // find a feasible starting point in [lb, 0]
+        start.clear();
+        for (UnsignedInteger k = 0; k < candidateNumber; ++ k)
+        {
+          Point candidate(sequence.generate());
+          for (UnsignedInteger j = 0; j < dimension; ++ j)
+            candidate[j] = candidate[j] * lb[j];
+          if (computePDF(candidate) > 0.0)
+          {
+            start = candidate;
+            break;
+          }
+        }
+        if (!start.getDimension())
+          throw InternalException(HERE) << "Could not find a feasible starting point to initialize ration of uniforms V inf";
+        problemVI.setBounds(Interval(lb, zero));
+        algo.setProblem(problemVI);
+        algo.setStartingPoint(start);
+        algo.run();
+        infV_[i] = -std::exp(algo.getResult().getOptimalValue()[0]);
+        LOGDEBUG(OSS() << "infV_[" << i << "]=" << infV_[i] << " v*=" << algo.getResult().getOptimalPoint());
+      }
+    }
```

Comment:
According to Codecov this whole block is untested: 

![image](https://github.com/user-attachments/assets/35901c09-34e8-428f-ae80-88776397a6e4)